src/main/scala/codacy/scalastyle/ScalaStyle.scala

```diff
@@ -32,7 +40,16 @@ object ScalaStyle extends Tool {
     }
   }
 
-  def parseToolResult(lines: List[String], path: Path): List[Result] = {
+  private lazy val configFileName = "scalastyle_config.xml"
+  private lazy val configFileNames = List(configFileName, "scalastyle-config.xml")
+
+  private lazy val defaultConfigFile: File = {
+    (better.files.File.root / "docs" / configFileName).toJava
```

Comment:
no, the file resides in the docker root. do you think it's better to use the one in the resources?